lib/src/quality/apps/text/links.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import requests
+from malevich.square import DF, Context, processor, scheme
+from pydantic import BaseModel
+
+
+@scheme()
+class Links(BaseModel):
+    link: str
+
+@processor()
+def asser_links(df: DF[Links], context: Context):
+    """
+        Check if the links are valid (do not return error status codes)
+
+        Input:
```

Comment:
Please, write

```markdown
## Input:

       (A) dataframe(s) that contains columns:
            - column_1: description
 ```
 
 This section is supposed to be displayed to no-code users on UI, so DF[Smth] is nothing for them

Reply:
Done